cypress/support/pages/account/pageCommonAccount.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+class Account_Common {
+
+  static #initialsButton = '[data-testid="initials"]';
+  static #settingsButton = '[data-testid="settings"]'
+
+  static #email = '[data-testid="user_email"]'
+  static #emailReadOnly = '[data-testid="user_email_readonly"]'
```

Comment:
I tested in my local but that `data-testid` was not present and test were failing at that point. 
Here are the screenshot from my local run:

<img width="1659" alt="Screenshot 2022-10-07 at 12 19 04" src="https://user-images.githubusercontent.com/97288209/194532210-373d0baf-e90e-4d74-ac9c-522824619a25.png">


<img width="1493" alt="Screenshot 2022-10-07 at 12 19 09" src="https://user-images.githubusercontent.com/97288209/194532168-e480697e-f0df-4091-8218-fc05be3cb147.png">

Reply:
Please see https://github.com/hpi-schul-cloud/schulcloud-client/pull/2983 . The id was added already, but not merged with main. You may want to test against the EW-246 (https://default-ew-246.cd.dbildungscloud.dev) branch instead of main ;)